### DISCUSSION TITLE
Option to display speed in metres per second, please

### DISCUSSION THREAD

**Discussion by ebenezergoose** [kylecorry31/Trail-Sense#1720] [New Feature Ideas]
Hi Kyle,

 I hope I am posting in the right place. I'm a big fan of TrailSense. However it lacks a feature that really all modern geographic / mereological software should have, that is the ability to display speed using SI base units. Quite a few reasons why I think you should consider this...

1. Metres per second is the correct and preferred SI (Système International d’Unités) way for displaying speed, as per General Conference on Weights and Measures (CGPM).
2. The **metre** and the **second** are SI "_base units_" the hour is not.
3. Metres per second is also the internationally preferred unit for expressing wind speed. 
4. Metres per second is standard speed measurement in almost all academic subjects, all around the planet, especially those involving physics. 
5. For many sporting purposes, mph and km/h are inappropriate. If one is say trying to run 100 metres in under ten seconds, one's speed in metres per second is much more useful measurement then either mph or km/h. Ditto for triathlon, speed walking, swimming etc.
6. It should be a relatively simple feature to add. Granted that is easy for a user to say. Nevertheless, a computer (smartphone) can easily multiply km/h by 5/18 (or even mph x 4/9, if you have to), whereas this is a much harder sum to have to do in one's head - especially if one is also trying to perform or monitor some sporting activity!

I would therefore be most grateful if you could give this your consideration for a future release please?

Many thanks and best wishes.

**Comment by kylecorry31** [comment]
Hi, thank you for the suggestion. I've created the following issue to track this: https://github.com/kylecorry31/Trail-Sense/issues/1721

The reason why I chose to display only mph and km/h is because it is more useful for the use cases of Trail Sense (https://github.com/kylecorry31/Trail-Sense/wiki/Use-Cases#trail-sense) than meters per second (or feet per second). For example, if I'm travelling at 4 km/h and the trail is 8 km long, I know it will take me about 2 hours - whereas if I displayed it in m/s it's harder to estimate longer distances. For short distances I agree that meters per second would be better, but that's typically outside Trail Sense's intended use cases. In either case, I'll add an option to choose the speed unit (defaulting to mph/kph based on the distance unit)

**Reply by ebenezergoose** [reply]
Thank you for giving this suggestion your consideration. Very much appreciated.  Your way of handling/displaying it seems eminently sensible too. I look forward to trying it for real. Please let me know if I can help testing etc. 

Many thanks and best wishes.
